test/data/manifests/rhel_91-x86_64-gce-boot.json

```diff
@@ -18,17 +18,19 @@
+        "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/el9/el9-x86_64-google-compute-engine-20220617",
+        "gpgkey": "",
```

Comment:
This test case manifest should be regenerated, because the GPG key from the `rhel-91` google repos definition was not removed in the end, however this manifest does not contain it. The reason is most probably that you generated it after you removed the GPG key from repo definitions, but then didn't do it again once you reverted the change.